Makefile

```diff
@@ -15,9 +15,22 @@ build: deps
 	docker build -t $(IMAGE_NAME):$(BUILD_VERSION) --build-arg python_version="3.6.3" -f docker/Dockerfile .
 
 .PHONY: test
-test: build
-	# Run Blueoil test
-	CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) bash ./blueoil_test.sh
+test: build test-classification test-object_detection test-semantic_segmentation
+
+.PHONY: test-classification
+test-classification: build
+	# Run Blueoil test of classification
+	CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) bash ./blueoil_test.sh  --task classification
+
+.PHONY: test-object_detection
```

Comment:
In `test-object_detection`, mixed "-" and "_" in one word looks not good.

Reply:
Thank you. I fixed to use `-` instead of `_`. :+1: